owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/shares/RemoveRemoteShareOperation.kt

```diff
@@ -46,63 +43,57 @@
  * @author David González Verdugo
  */
 
-public class RemoveRemoteShareOperation extends RemoteOperation {
-
-    private static final String TAG = RemoveRemoteShareOperation.class.getSimpleName();
-
-    private int mRemoteShareId;
-
-    /**
-     * Constructor
-     *
-     * @param remoteShareId Share ID
-     */
-
-    public RemoveRemoteShareOperation(int remoteShareId) {
-        mRemoteShareId = remoteShareId;
-    }
+class RemoveRemoteShareOperation
+/**
+ * Constructor
+ *
+ * @param remoteShareId Share ID
+ */
+    (private val mRemoteShareId: Long) : RemoteOperation<ShareParserResult>() {
```

Comment:
converted file?? you have only one constuctor and java doc should be above class declaration

Reply:
How can I include java doc for a constructor if is part of the class declaration in Kotlin instead of a separate method? 🤦‍♂️

Reply:
I can put it above but is pretty ugly

![screenshot 2019-03-05 at 17 40 05](https://user-images.githubusercontent.com/15261363/53821289-c1633400-3f6d-11e9-9e0b-b36146fa12ca.png)